### ISSUE TITLE
Evaluation mode config

### ISSUE THREAD

**Issue by solidexu** [voldemortX/pytorch-auto-drive#79]
Hello,
Could you please share your config that can record best model during training? it seems the mode is closed due to 'get validation set ' function always return None

**Comment by voldemortX** [comment]
@solidexu In lane detection, we do not use validation for possible early stopping. Since almost everyone don't use online validation in this field, and in earlier formulation of this framework, we observe that the final model is usually the best.
